editor/project_export.cpp

```diff
@@ -830,6 +830,17 @@ void ProjectExportDialog::_refresh_parent_checks(TreeItem *p_item) {
 }
 
 void ProjectExportDialog::_export_pck_zip() {
+	Ref<EditorExportPreset> current = get_current_preset();
+	ERR_FAIL_COND(current.is_null());
+
+	String p_path = current->get_export_path();
+	if (p_path != "") {
+		int pos = MAX(p_path.rfind("/"), p_path.rfind("\\"));
+		if (pos != -1) {
+			String dir = p_path.substr(0, pos);
+			export_pck_zip->set_current_dir(dir);
+		}
+	}
```

Comment:
Isn't `p_path.get_base_dir()` sufficient?

Reply:
Yep, you're right. That would be simpler.